app/(main)/register/page.tsx

```diff
@@ -115,6 +119,7 @@ const Register = (): JSX.Element => {
         />,
       );
     } catch (error) {
+      setIsLoading(false);
```

Comment:
Could you please explain why `setIsLoading(false)` is used twice? One in the `try` part and in the `catch`.

Reply:
> Could you please explain why `setIsLoading(false)` is used twice? One in the `try` part and in the `catch`.

@vmaineng this was from @Clue355's branch of this branch that was approved and merged in.

Reply:
Though I'd like to make this as similar to my code on the login page as possible.

@shashilo @Clue355 @vmaineng @chris-nowicki thoughts on me changing the `onSubmit` on the `register/page.tsx` to be inline with the one on the `login/page.tsx`?

**Current Login `onSubmit`**
```tsx
const onSubmit: SubmitHandler<LoginUserSchemaType> = async (
    data: LoginUserSchemaType,
  ): Promise<void> => {
    try {
      setIsLoading(true);
      await login(data);
    } catch (error) {
      console.error(error);
      throw new Error('An error occurred while logging in');
    } finally {
      setIsLoading(false);
    }
  };
```

**Current Register `onSubmit`**
```tsx
const onSubmit: SubmitHandler<RegisterUserSchemaType> = async (
    data: RegisterUserSchemaType,
  ): Promise<void> => {
    try {
      setIsLoading(true);
      await registerAccount(data);
      await login(data);
      setIsLoading(false);
      router.push('/league/all');
      toast.custom(
        <Alert
          variant={AlertVariants.Success}
          message="You have successfully registered your account."
        />,
      );
    } catch (error) {
      setIsLoading(false);
      console.error('Registration Failed', error);
      toast.custom(
        <Alert variant={AlertVariants.Error} message="Something went wrong!" />,
      );
    }
  };
```

Reply:
@shashilo @Clue355 @vmaineng @chris-nowicki Please advise on the comment above.

Reply:
@ryandotfurrer in what ways to you plan to make it inline? by adding a finally to the register try/catch?

Reply:
> @ryandotfurrer in what ways to you plan to make it inline? by adding a finally to the register try/catch?

@chrisnowicki precisely. Just to make them consistent.

Reply:
Use finally. It allows you to only call the setIsLoading false once.

Reply:
> Use finally. It allows you to only call the setIsLoading false once.

@shashilo added in the latest commit